src/main/java/com/softserve/sportshub/comment/controller/CommentController.java

```diff
@@ -0,0 +1,68 @@
+package com.softserve.sportshub.comment.controller;
+
+import com.softserve.sportshub.comment.dto.CommentDto;
+import com.softserve.sportshub.comment.dto.CreateCommentDto;
+import com.softserve.sportshub.comment.dto.ReadCommentDto;
+import com.softserve.sportshub.comment.service.CommentService;
+import com.softserve.sportshub.comment.domain.Comment;
+import com.softserve.sportshub.user.UserService;
+import com.softserve.sportshub.user.dto.UserDto;
+import com.softserve.sportshub.utils.JwtUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
+public class CommentController {
+
+
+    public CommentService service;
+    public UserService userService;
+
+    public CommentController(CommentService service, UserService userService) {
+        this.service = service;
+        this.userService = userService;
+    }
+
+    @GetMapping("/comments")
+    public ResponseEntity<List<ReadCommentDto>> getAllComments() {
+        return ResponseEntity.ok().body(service.list());
+    }
```

Comment:
It's not a mistake, but you could return the list directly here, without wrapping it on ResponseEntity. The output will be the same.